base/floatfuncs.jl

```diff
@@ -325,7 +325,7 @@ const ≈ = isapprox
 
 This is equivalent to `!isapprox(x,y)` (see [`isapprox`](@ref)).
 """
-≉(args...; kws...) = !≈(args...; kws...)
+≉(args...; kw...) = !≈(args...; kw...)
```

Comment:
I find the singular `kw` odd: by the reasoning that `kws` should be `kw`, it seems `args` should be `arg` here for instance. In both cases, however, the absence of 's' suggests the quantity is singular rather than plural (contrary to intention: i.e., the `...` is there to allow multiple arguments).

Reply:
Aye, and that is supported by the observation that In a couple of cases I had to leave `kws...` as is because of a subsequent `for kw in kws` loop.

I think it may be a good idea to switch from `kw` to `kws` in most cases. (though keep `kw...` when there can be at most one keyword argument)

I just noticed that [Blue](https://github.com/invenia/BlueStyle) recommends `kwargs...`, an even more verbose option that is quite popular in the stdlibs already:
```
grep -owr 'kwargs\.\.\.' stdlib | wc -l
     499
```
But only slightly more common than `kw` in base:
```
x@X julia_dm % grep -owr 'kwargs\.\.\.' base | wc -l            
      88
x@X julia_dm % grep -owr 'kws\.\.\.' base | wc -l 
      13
x@X julia_dm % grep -owr 'kw\.\.\.' base | wc -l 
      77
```

Reply:
> I just noticed that [Blue](https://github.com/invenia/BlueStyle) recommends kwargs..., an even more verbose option that is quite popular in the stdlibs already:

I think `kws` is better.

Reply:
There seems to be a solid consensus for `kws` instead of `kw` backed by sensible reasoning. I'm going to reverse this PR to change `kw` to `kws`.